src/BenchmarkDotNet.Runtime.Classic/InProcessExecutor.cs

```diff
@@ -101,13 +101,13 @@ private int ExecuteCore(IHost host, Benchmark benchmark, ILogger logger)
             var affinity = benchmark.Job.ResolveValueAsNullable(EnvMode.AffinityCharacteristic);
             try
             {
-                process.TrySetPriority(ProcessPriorityClass.High, logger);
+                ProcessExtensions.TrySetPriority(process, ProcessPriorityClass.High, logger);
 #if CLASSIC
```

Comment:
Could we completely remove the `#if CLASSIC` line here and hide them inside some portability API?